guides/common/modules/proc_configuring-project-settings-for-keycloak-authentication-using-the-web-ui.adoc

```diff
@@ -0,0 +1,26 @@
+[id="configuring-project-settings-for-keycloak-authentication-using-the-web-ui_{context}"]
+= Configuring {Project} Settings for {Keycloak} Authentication Using the Web UI
+
+Use this procedure to configure {Project} settings for {Keycloak} authentication using the {Project} web UI.
+
+.Prerequisite
+
+* Ensure that the *Access Type* setting in the {Project} client in the {Keycloak} web UI is set to *confidential*
+
+.Procedure
+
+. In the {Project} web UI, navigate to *Administer* > *Settings*, and click the *Authentication* tab.
+. Locate the *Authorize login delegation* row, and in the *Value* column, set the value to *Yes*.
+. Locate the *Authorize login delegation auth source user autocreate* row, and in the *Value* column,
+set the value to *External*.
+. Locate the *Login delegation logout URL* row, and in the *Value* column, set the value to
+*https://{foreman-example-com}/users/extlogout*.
+. Locate the *OIDC Algorithm* row, and in the *Value* column, set the algorithm for encoding on {Keycloak} to *RS256*.
+. Locate the *OIDC Audience* row, and in the *Value* column, set the value to the client ID for {Keycloak}.
```

Comment:
not sure but missing e.g. here as `['satellite.example.com-foreman-openidc'].`

Reply:
We are not mentioning the value here, we are just indicating what needs to go in the setting.

Reply:
I expect that users are able to just copy past those values from the RHSSO web UI. This is a fool-proof method.

Reply:
Okay